Makefile

```diff
@@ -15,7 +15,14 @@ tools.airbyte-ci.check: # Check if airbyte-ci is installed correctly
 tools.airbyte-ci.clean: ## Clean airbyte-ci installations
 	@./airbyte-ci/connectors/pipelines/pipelines/external_scripts/airbyte_ci_clean.sh
 
-tools.install: tools.airbyte-ci.install tools.airbyte-ci.check
 
-.PHONY: tools.install tools.airbyte-ci.install tools.airbyte-ci-dev.install tools.airbyte-ci.check tools.airbyte-ci.clean
+tools.pre-commit.install: tools.airbyte-ci.install ## Install pre-commit hooks
+	@echo "Installing pre-commit..."
+	@brew install pre-commit
+	@echo "Installing pre-push hooks..."
+	@pre-commit install --hook-type pre-push
+	@echo "Pre-commit installation complete and pre-push hooks installed."
```

Comment:
This is because my global hooks path is overridden by something in the platform-internal repo for FE hooks: 
```
[core]
        repositoryformatversion = 0
        filemode = true
        bare = false
        logallrefupdates = true
        ignorecase = true
        precomposeunicode = true
        hooksPath = airbyte-webapp/.husky
[remote "origin"]
        url = git@github.com:airbytehq/airbyte.git
        fetch = +refs/heads/*:refs/remotes/origin/*
```
So this will likely fail for anyone who has contributed to the frontend. Related issue [here](https://github.com/pre-commit/pre-commit/issues/1198)

Reply:
IMO it's likely an issue with how the platform one is set. Global seems like too wide of a net

Reply:
Oh, perfect! Glad I happened to be an edge case that's probably common around here, lol